src/bundler.ts

```diff
@@ -188,8 +188,9 @@ export class Bundler {
       bundleImports?: Set<string>): Promise<ASTNode> {
     const docUrl = docBundle.url;
     const document = await this._prepareBundleDocument(docBundle);
+
     this._appendHtmlImportsForBundle(document, docBundle);
-    importUtils.rewriteImportedUrls(document, docUrl, docUrl);
+    importUtils.debaseDocument(docUrl, document);
```

Comment:
If its okay, I've decided to rename it `rewriteDocumentToEmulateBaseTag`.  It actually applies two effects.  One is that it adjusts the urls accordingly.  Two is that it applies the "target" attribute of the base tag to links and forms which don't have a target.  I also added a regression test to ensure isolation of the effects of target attribute application.